gadget/install/install.go

```diff
@@ -96,22 +96,26 @@ func saveStorageTraits(mod gadget.Model, allLaidOutVols map[string]*gadget.LaidO
 	return nil
 }
 
-func maybeEncryptPartition(part *gadget.OnDiskStructure, encryptionType secboot.EncryptionType, sectorSize quantity.Size, perfTimings timings.Measurer) (fsParams *mkfsParams, encryptionKey keys.EncryptionKey, err error) {
+func maybeEncryptPartition(odls *onDiskAndLaidoutStructure, encryptionType secboot.EncryptionType, sectorSize quantity.Size, perfTimings timings.Measurer) (fsParams *mkfsParams, encryptionKey keys.EncryptionKey, err error) {
 	mustEncrypt := (encryptionType != secboot.EncryptionTypeNone)
-	partDisp := roleOrLabelOrName(part)
+	onDisk := odls.onDisk
+	laidOut := odls.laidOut
+	partDisp := roleOrLabelOrName(laidOut.Role, onDisk)
```

Comment:
looking closer it seems partDisp here can be removed, as it's used only below where laidout.Role could be used directly instead as it won't be "" for sure